packages/@okta/vuepress-site/standards/SAML/setting_up_a_saml_application_in_okta/index.md

```diff
@@ -18,14 +18,14 @@ SAML Identity Provider or "SAML IdP"), with the details of your application
 Here is how to set up a SAML application in Okta:
 
 > If you are using the developer dashboard you will first need to switch to the Classic UI. <br />
-If you see a <b>< > Developer</b> prompt in the top left, click it and select <b>Classic UI</b> to switch to the Classic UI. 
+If you see a <b>< > Developer</b> prompt in the top left, click it and select <b>Classic UI</b> to switch to the Classic UI.
```

Comment:
What is the extra `< >` there doing?

Reply:
I don't know why that was put there.... odd... all i did was remove a `"`